Sources/AirbnbSwiftFormatTool/airbnb.swiftformat

```diff
@@ -102,4 +102,5 @@
 --rules wrapMultilineConditionalAssignment
 --rules blankLineAfterMultilineSwitchCase
```

Comment:
`blankLineAfterMultilineSwitchCase` - Why can't I find this in either
- https://github.com/nicklockwood/SwiftFormat/blob/main/Rules.md or
- https://github.com/nicklockwood/SwiftFormat/blob/main/Sources/Rules.swift
??

Reply:
Some rules are still only available on the `develop` branch. You can see info about them in the `Rules.md` file on the `develop` branch: https://github.com/nicklockwood/SwiftFormat/blob/develop/Rules.md

SwiftFormat development happens on the `develop` branch. The `main` branch is only updated when releasing new versions.

Reply:
thanks!